src/utils/accessToken.ts

```diff
@@ -96,6 +96,74 @@ export const accessToken = {
     return userId;
   },
 
+  getAppIdFromAccessToken(accessToken: string): string {
+    let appId: string = '';
+
+    if (!accessToken || accessToken.length === 0) {
```

Comment:
Let's centralize the logic to parse a token to avoid repetition

Reply:
Good point 👍